rcode-reader/src/main/java/eu/rssw/pct/elements/DataRelationElement.java

```diff
@@ -0,0 +1,52 @@
+package eu.rssw.pct.elements;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+import eu.rssw.pct.RCodeInfo;
+
+public class DataRelationElement extends ClassElement {
+  private int pairCount;
+  private int flags;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "flags" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1068)